test/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ itk_add_test(NAME TwoProjection2D3DRegistrationDownSizedCTTest
     DATA{Input/BoxheadCT.img,BoxheadCT.hdr}
   )
 
+if(0) #HACK File missing from download
```

Comment:
@dzenanz  I don't think we should use this PR, until the tests are fixed.  Currently the data is not available for download.  How can we restore these data sets?